lang/c++/impl/Compiler.cc

```diff
@@ -144,6 +152,19 @@ const int64_t getLongField(const Entity& e, const Object& m,
     return it->second.longValue();
 }
 
+// Unescape double quotes (") for de-serialization.  This method complements the
+// method NodeImpl::escape() which is used for serialization.
+static void unescape(std::string& s) {
+  boost::replace_all(s, "\\\"", "\"");
```

Comment:
We consistently use four spaces for indentation. This and the following function use two spaces.